src/main/java/seedu/address/ui/PersonListPanel.java

```diff
@@ -34,6 +34,8 @@ public PersonListPanel(ObservableList<Person> personList) {
         protected void updateItem(Person person, boolean empty) {
             super.updateItem(person, empty);
 
+            logger.info("people have ");
```

Comment:
erm.. what is this for?

Reply:
Sorry I was testing!! Thanks for noticing!